CocoaPods.md

```diff
@@ -72,18 +72,31 @@ my-plugin/
         └── Podfile
 ```
 
+Podfile:
+```
+pod 'GoogleMaps'
+```
+
+## Install the Plugin
+
 Next, install the plugin:
 
-```
+```bash
 tns plugin add ../my-plugin
 ```
 
-## Build the project
+> **NOTE:** Installing CocoaPods sets the deployment target of your app to iOS 8, if not already set to iOS 8 or later. This change is required because CocoaPods are installed as shared frameworks to ensure that all symbols are available at runtime.
 
-```
+## Build the Project
+
+```bash
 tns build ios
 ```
 
-This modifies the `MYCocoaPods.xcodeproj` and creates a workspace with the same name.
+This modifies the `MYCocoaPodsApp.xcodeproj` and creates a workspace with the same name.
+
+> **IMPORTANT:** You will no longer be able to run the `xcodeproj` alone. NativeScript CLI will build the newly created workspace and produce the correct package.
+
+## Troubleshooting
 
-> **IMPORTANT:** You will no longer be able to run the `xcodeproj` alone. NativeScript CLI will build the newly created workspace and produce the correct .ipa.
\ No newline at end of file
+If there are linker errors after building, you may have to link with some native frameworks described in the CocoaPod documentation. See [`build.xcconfig` file](PLUGINS.md#buildxcconfig-specification) specification on how to do this.
```

Comment:
In case of post-build linker errors, you might need to resolve missing dependencies to native frameworks required by the installed CocoaPod. For more information about how to create the required links, see  the [`build.xcconfig` specification](PLUGINS.md#buildxcconfig-specification).